accounts/tests.py

```diff
@@ -49,4 +49,19 @@ def test_login_template(self):
         self.assertTemplateUsed(self.response, 'account/login.html')
         self.assertContains(self.response, 'Sign In')
         self.assertNotContains(
-            self.response, 'Hi there! I should not be on the page.')
\ No newline at end of file
+            self.response, 'Hi there! I should not be on the page.')
+
+class LogoutTests(TestCase):
+
+    """
+    As a logged-in user I would like a logout link so that I can
+    log out of my account
+    """
+
+    def setUp(self):
```

Comment:
I don't see a logged in user here?  How do we know this is getting a 302 because a user is logging out vs just redirecting because there was no logged in user to being with?  Also, we can test which templates were used, which may be helpful here.

Reply:
I changed the test so now, if a user is not logged in, they are redirected to the landing page template, and if they are logged in, they get the logout template.